packages/hoprd/src/index.ts

```diff
@@ -211,6 +205,16 @@ async function main() {
       process.exit(1)
     }
   }
+
+  async function stopGracefully(signal) {
```

Comment:
presumably can't be an async function for process.on?

Reply:
Why not?